guide/react/list-other.mdx

```diff
@@ -0,0 +1,42 @@
+---
+sidebar_position: 4
+---
+
+# Display other participants' media
+
+To access data and media of other participants, use the `useParticipants` hook.  
+It returns two properties, `participants` and `localParticipant`.  
+They contain all tracks of other participants and all tracks of the local user, respectively.
+
+### Example of playing all available media
+
+```tsx
+import { useParticipants } from "@fishjam-cloud/react-client";
+
+function Component() {
+  const { localParticipant, participants } = useParticipants();
+
+  const allParticipants = [localParticipant, ...participants];
+
+  return (
+      <ul>
+        {allParticipants.map((p) => {
+          const videoTracks = [...p.videoTracks, ...p.screenshareVideoTracks];
+          const audioTracks = [...p.audioTracks, ...p.screenshareAudioTracks];
+
+          return (
+              <li key={p.id}>
```

Comment:
I think `VideoRenderer` should be wrapped in `li`. As if there is only audio track, then it would create empty list element.

Reply:
I wanted this code to be simple as possible and show the user how the data can be retrieved, not necessarily used. The `VideoRenderer` is just a hint for the user he needs to handle the data somehow.

Reply:
I'm 50/50 here, but let it be as is.